### DISCUSSION TITLE
encountered issue on plugin use

### DISCUSSION THREAD

**Discussion by balajikhandekar** [QuickSander/homebridge-http-rgb-push#6] [Q&A]
Hi,

Before i post my issue i must say this is awesome plugin with Apple homekit works fine.

The only issue I see is when the light turned on and when reopen Ios Home app or when i run accessory from homebridge local portal i get following error-

... hue is currently NaN. RGB: 
[6/9/2021, 10:06:29 AM] [homebridge-http-rgb-push] This plugin generated a warning from the characteristic 'Hue': characteristic value expected valid finite number and received "NaN" (number). See https://git.io/JtMGR for more info.

similarly for saturation i get same error.

Is this something known issue or if you could provide quick fix for this?

Thank you.

**Comment by QuickSander** [comment]
Hi, I do recognize this. I have to look into this. I believe it had to do with when you do not use notification server. Since then the variable is uninitialized. I believe I had a fix in my pipeline.

**Reply by balajikhandekar** [reply]
Hi, 
Thanks for your reply.
Not sure i will be able to use notification server i am using nodemcu to control LED.
Thank you

**Reply by QuickSander** [reply]
This logging is only printed when HomeKit does a getHue request. According to the logging, at that moment the RGB string returned by your device is an empty string ("RGB: <nothing>"). 

Could you verify that making a get request to your device to get the current RGB value returns a hexadecimal string? 

What is your configuration for this plugin (json)?
